docs/source/quick_start.rst

```diff
@@ -114,6 +523,7 @@ install it separately on another machine as well. Here is how you would use
    possible.
 
 
+
```

Comment:
The below PHOENIX should be followed by `-----` for it to be a section like the above sections. Please see `quick_start.rst` file in GitHub web interface to realize my comment.